pages/shoppingCart.js

```diff
@@ -1,10 +1,12 @@
+import styled from "styled-components";
 import { useRouter } from "next/router";
 import { useEffect, useState } from "react";
 import Image from "next/image";
 
 export default function ShoppingCart({ ticket }) {
   const router = useRouter();
   const [cartTickets, setCartTickets] = useState([]);
```

Comment:
You need to use the `useLocalStorageState` here. Currently, your shopping cart is always empty because you are using local state with an initially empty array.